cmd/transflect-operator/metrics.go

```diff
@@ -22,10 +23,79 @@ var (
 	)
 	preprocessErrCounter = promauto.NewCounter(prometheus.CounterOpts{
 		Name: "transflect_preprocess_error_total",
-		Help: "The total number of replicaset ids popped off the queue that errored before being processed",
+		Help: "The total number of replicaset ids popped off the queue that errored before being processed.",
 	})
 	ignoredCounter = promauto.NewCounter(prometheus.CounterOpts{
 		Name: "transflect_ignored_total",
-		Help: "The total number of replicaset ids popped off the queue that were ignored and didn't need processing",
+		Help: "The total number of replicaset ids popped off the queue that were ignored and didn't need processing.",
 	})
+
+	// queue metrics.
+	qDepthGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "transflect_queue_depth",
+		Help: "Current depth of the work queue.",
+	}, []string{"queue_name"})
```

Comment:
I dont think this queue name label is useful unless you name your queue. I suggest you use `NewNamedRateLimitingQueue()` instead of `NewRateLimitingQueue()` and name the queue `replicasets`.